core/heap.c

```diff
@@ -3374,6 +3418,7 @@ common_global_heap_alloc(thread_units_t *tu, size_t size HEAPACCT(which_heap_t w
     acquire_recursive_lock(&global_alloc_lock);
     p = common_heap_alloc(tu, size HEAPACCT(which));
     release_recursive_lock(&global_alloc_lock);
+
```

Comment:
Forgot to revert this and the one below?  Unrelated.

Reply:
Done